From d73c2d31f54ca32a742bfcd4bedf85fb640ee4e7 Mon Sep 17 00:00:00 2001
From: "Derek J. Clark" <derekjohn.clark@gmail.com>
Date: Wed, 26 Aug 2026 01:56:07 +0000
Subject: [PATCH 112/173] Revert "[NOT-FOR-UPSTREAM] HID: hid-msi: Report RGB
 LED under the Legion Go name"

This reverts commit 54ef0293719b8443cb0d981e2caad14a62e7793f.
---
 drivers/hid/hid-msi.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/hid/hid-msi.c b/drivers/hid/hid-msi.c
index f74856d4e452..d4688caef296 100644
--- a/drivers/hid/hid-msi.c
+++ b/drivers/hid/hid-msi.c
@@ -1512,7 +1512,7 @@ static int claw_probe(struct hid_device *hdev, u8 ep)
 	if (!drvdata->bmap_support)
 		dev_dbg(&hdev->dev, "M-Key mapping is not supported. Update firmware to enable.\n");
 
-	drvdata->led_mc.led_cdev.name = "go:rgb:joystick_rings";
+	drvdata->led_mc.led_cdev.name = "msi_claw:rgb:joystick_rings";
 	drvdata->led_mc.led_cdev.brightness = 0x50;
 	drvdata->led_mc.led_cdev.max_brightness = 0x64;
 	drvdata->led_mc.led_cdev.color = LED_COLOR_ID_RGB;
-- 
2.55.0

